src/main/java/com/brightpath/learnify/persistance/common/PersistentMapper.java

```diff
@@ -43,29 +45,43 @@ public Note asNote(NoteEntity entity) {
         );
     }
 
-    public QuizSimpleResult asBestSimpleResult(QuizEntity entity) {
-        if (entity.getBestNumberOfIncorrect() == null || entity.getBestNumberOfCorrect() == null) {
+    public QuizSimpleResult asBestSimpleResult(QuizEntity entity, String userId) {
```

Comment:
Nitpick: 
I think this shouldn't be here as this class is just for mappings between domain nad persistance objects (but as long as we don't care too much about code quality you don't have to waste time for that - saying just as information for future changes)